packages/lib-classifier/src/store/feedback/helpers/get-feedback-from-tasks.js

```diff
@@ -1,17 +1,18 @@
-import _ from 'lodash'
-
 // Filters a workflow's tasks object to an object of `taskId: [feedback rules]` pairs
-function getFeedbackFromTasks (tasks) {
-  return _.reduce(tasks, (result, task, taskId) => {
-    if (_.get(task, 'feedback.enabled', false) &&
-      _.get(task, 'feedback.rules', []).length > 0) {
-      return _.assign({}, result, {
-        [taskId]: task.feedback.rules
-      })
-    } else {
-      return result
+function getFeedbackFromTasks (tasks = {}) {
+  const result = {}
+  Object.entries(tasks).forEach(function getTaskRules ([taskKey, task]) {
```

Comment:
If we want to support IE11, then this should be changed to use `Object.keys()`.

Reply:
We're not supporting IE11 in the rebuild: https://github.com/zooniverse/front-end-monorepo/blob/master/docs/arch/adr-4.md

Reply:
And fwiw, Microsoft is dropping legacy support for it in 2020 and will make Edge available for Win 7 users to download.